scripts/python/golioth/README.md

```diff
@@ -0,0 +1,100 @@
+# Golioth client Python library
+
+This is a basic Golioth client library.
+
+## Install
+
+Just install this package from source with following command:
+
+``` sh
+pip install .
+```
+
+If you plan on editing / development of this package, consider using `-e` (`--editable`) parameter
+when installing it.
+
+## CLI
+
+This package installs with CLI script called `golioth`. There are various commands implemented,
+which can be listed by using `--help` argument such as:
+
+``` sh
+$ golioth --help
+Usage: golioth [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --config-path PATH  Path to goliothctl configuration
+  --help              Show this message and exit.
+
+Commands:
+  call      Call RPC method on device.
+  logs      Logging service related commands.
+  settings  Settings service related commands.
+```
+
+or for each subcommand like `golioth logs --help`.
+
```

Comment:
Can you mention the local configuration option you shared with me, about creating the `.golioth-local.yaml` file? Also, it's probably a good idea to add that file to `.gitignore`.

Reply:
```
golioth --help
```
mentions:
```
Options:
  --config-path PATH  Path to goliothctl configuration
```

Reply:
Yeah, I was thinking of adding something just a convenience to people like me :)

For instance, I was getting a cryptic python error, and didn't know I should be using this option at all. I also was not familiar with what all is entailed in goliothctl configuration (e.g. the file type of YAML and the minimal config options I needed to set).